### DISCUSSION TITLE
Claim a squatted username since 2012

### DISCUSSION THREAD

**Discussion by Naushikha** [community/community#21598] [New to GitHub]
<p>Hi,</p>
<p>I am trying to create an account for an open source software suite my university is making. But our software name is already taken.<br>
I think the account fits on the Name Squatting policy since the account was never used since 2012.<br>
I have opened a ticket on support, but didnt recieve any update there for a while now.</p>
<p>Can someone help me with this?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>We do sometimes evaluate username requests but because these are not blocking issues, they are not prioritised. This means our support team will be working on tickets relating to billing and login and account reviews before they look at username requests. The wait on these can unfortunately be a few weeks depending on how busy the priority queues are.</p>
<p>Please also remember that not all GitHub activity is public, so just because a profile looks empty, doesn’t mean it is. If you are in a hurry, the best thing would be to choose an available organisation name.</p>

**Reply by arifpro** [reply]
@canuckjacq I have a domain devarif.me and I want to change my Github username by devarif/mearif, but it's not available and I also checked that that user is not acitve for long long time. any idea what should I do?

**Comment by Naushikha** [comment]
<p>Thank you so much for the reply <a class="mention" href="/canuckjacq">@canuckjacq</a> . :grinning_face_with_smiling_eyes: This was a much needed clarification for me.<br>
Honestly, I had no idea that there are different priorities on tickets.<br>
There is no hurry on this, I will surely wait until the team is free to take on my request.</p>
